lib/util/yaml.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  DumpOptions,
+  LoadOptions,
+  loadAll as multiple,
+  load as single,
+  dump as upstreamDump,
+} from 'js-yaml';
+import { regEx } from './regex';
+
+interface YamlOptions extends LoadOptions {
+  removeTemplates?: boolean;
+}
+
+export function loadAll(
+  content: string,
+  iterator?: null | undefined,
+  options?: YamlOptions,
+): unknown[] {
+  const massagedContent = massageContent(content, options);
+
+  return multiple(massagedContent, iterator, options);
+}
+
+export function load(content: string, options?: YamlOptions): unknown {
```

Comment:
can we rename it as load is really common. I think we should match the JSON parseJson